layer1/bridge/src/inbox_sc.mligo

```diff
@@ -6,20 +6,26 @@
 
 type message = Inbox.message
 
-(**The whole inbox, the historic of all the inboxes since level 0 of the rollup**)
-type messages = (nat, message list) big_map
+type inbox_level = nat
+
+(** Each [inbox_level] has its own inbox, which is a list of messages.
+    The [inbox_level] is used in chain_sc and starts from 0.
+    The inbox level 0 belongs to the genesis block so its inbox
+    should have no message. *)
+type inboxes = (inbox_level, message list) big_map
```

Comment:
Non blocking: I'm not sure `inboxes` is a clear name, there is only one inbox, so why the plural ?

Reply:
I was thinking about `inbox = message list` so `(inbox_level, inbox) big_map`.
otherwise, I think it would be `inbox = (messages_level, message list) big_map`.
I'm going to merge it, we can change the name in next PR.